src/graphalgorithm.cpp

```diff
@@ -0,0 +1,2 @@
+#include <sealib/graphalgorithm.h>
```

Comment:
Note: Move the contents of the header here.

Reply:
graphalgorithm is deprecated, the updated version is eulertrail.h/cpp

Reply:
graphalgorithm is deprecated, the updated version is eulertrail.h/cpp